system-tests/.env.example

```
@@ -0,0 +1,5 @@
+CONSUMER_KEY_VAULT=kv257company1
+PROVIDER_KEY_VAULT=kv257company2
+EDC_VAULT_CLIENTID=16437dd0-40ab-42f9-a099-6c326df06203
+EDC_VAULT_TENANTID=72f988bf-86f1-41af-91ab-2d7cd011db47
+EDC_VAULT_CLIENTSECRET=<secret>
```

nit: adding places holder as text explanation.
```suggestion
CONSUMER_KEY_VAULT=<consumer-key-vault-name>
PROVIDER_KEY_VAULT=<provider-key-vault-name>
EDC_VAULT_CLIENTID=<client-id-of-application's-service-principal-object>
EDC_VAULT_TENANTID=<azure-ad-tenant-id>
EDC_VAULT_CLIENTSECRET=<client-secret-of-application's-service-principal-object>
```